Packages/com.unity.inputsystem/CHANGELOG.md

```diff
@@ -44,6 +44,7 @@ however, it has to be formatted properly to pass verification tests.
 - Fixed InputManager.asset file growing in size on each Reset call.
 - Fixed Opening InputDebugger throws 'Action map must have state at this point' error
 - Fixed Cut/Paste behaviour to match Editor - Cut items will now be cleared from clipboard after pasting.
+- Fixed Pasting items between windows not working until an item in the Action Map/Action lists was selected.
```

Comment:
Would be good to be more specific and clarify that "windows" here refer to "Input Action Editor windows". Also good to be more specific about "not working", instead suggest mentioning what would happen.